semver/semver_test.go

```diff
@@ -281,6 +283,37 @@ func TestJSON(t *testing.T) {
 	}
 }
 
+func TestYAML(t *testing.T) {
+	document, err := yaml.Marshal(fixtures)
```

Comment:
Default YAML marshalling example:

```
- greaterversion: 1.0.0-alpha.beta
  lesserversion: 1.0.0-alpha.1
- greaterversion: 1.0.0-alpha.1
  lesserversion: 1.0.0-alpha
```

We can implement `MarshalYAML() (interface{}, error)` if we want, but no Ignition tools need marshaling at the moment.